packages/service-library/src/servicelib/fastapi/long_running_tasks/_context_manager.py

```diff
@@ -87,7 +88,7 @@ async def periodic_task_result(
     - `status_poll_interval` optional: when waiting for a task to finish,
         how frequent should the server be queried
 
-    raises: `TaskClientResultError` if the task finished with an error instead of
+    raises: `Exception` if the task finished with an error instead of
```

Comment:
Exception?? I see at least `TaskClientTimeoutError` below, which contradict what you have on line 93
maybe you should rephrase in `raises whatever Exception the task might raise` ?

Reply:
TaskClientTimeoutError